contrib/intel/jenkins/Jenkinsfile

```diff
@@ -921,7 +921,7 @@ pipeline {
       }
       dir ("${DELETE_LOCATION}/middlewares") { deleteDir() }
     }
-    cleanup {
+    failure {
```

Comment:
I am confused. The title says "do not delete logs on failure" but the body and the code here suggest "delete logs on failure".

Reply:
You're right I am changing it now.